app/controllers/StartDeclarationController.scala

```diff
@@ -35,7 +37,13 @@ class StartDeclarationController @Inject()(
   )(implicit appConfig: FrontendAppConfig) extends LocalBaseController {
 
   def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
-    Future.successful(Ok(view()))
+    val allSectionsCompleted =
```

Comment:
using this piece of code in more than one place so plz move this code to LocalBaseController